docs/solution/longterm/index.md

```diff
@@ -0,0 +1,69 @@
+(longterm-store)=
+(timeseries-longterm)=
+(timeseries-long-term-storage)=
+
+# Long-term store
+
+:::{div} sd-text-muted
+Never retire data just because your other systems can't handle the cardinality.
+:::
+
+CrateDB stores large volumes of data, keeping it accessible for querying
+and insightful analysis, even considering historic data records.
+
+:::{todo}
+**Instructions:**
+Elaborate a bit longer about the topic domain and the ingredients of this section
+in an abstract way, concisely highlighting and summarizing relevant benefits,
+like the `../analytics/index` and `../industrial/index` pages are doing it already.
+:::
```

Comment:
@coderabbitai: Let me repeat my instructions and better reference corresponding pages here, in Markdown format, so you might be able to use them better for coming up with relevant improvements.

> Elaborate a bit longer about the topic domain and the ingredients of this section in an abstract way, concisely highlighting and summarizing relevant benefits, like the [Real-time raw-data analytics] and [Industrial data] landing pages are doing it already.

[Real-time raw-data analytics]: https://github.com/crate/cratedb-guide/raw/refs/heads/main/docs/solution/analytics/index.md
[Industrial data]: https://github.com/crate/cratedb-guide/raw/refs/heads/main/docs/solution/industrial/index.md